apps/manager_app/manager_api.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file    manager_api.h
+ * @author  Cypherock X1 Team
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ */
+#ifndef MANAGER_API_H
+#define MANAGER_API_H
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+#include "manager_app.h"
```

Comment:
We are expecting to include `manager_api.h` in `manager_app.h`, so it will be a cyclic include.
Please fix this by including required headers directly.

Reply:
updated